framework/src/csg/CSGRegion.C

```diff
@@ -33,9 +33,11 @@ CSGRegion::CSGRegion(const CSGRegion & region_a,
 {
   _region_type = region_type;
   if (_region_type != "INTERSECTION" && _region_type != "UNION")
```

Comment:
```suggestion
  if (_region_type != RegionType::INTERSECTION && _region_type != RegionType::UNION)
```

I think there are if-statements throughout that you would want to update still right?